.travis.yml

```diff
@@ -3,36 +3,37 @@ language: php
 sudo: false
 
 cache:
-    directories:
-        - vendor
-        - $HOME/.composer/cache/files
+  directories:
+    - vendor
+    - $HOME/.composer/cache/files
```

Comment:
Can you revert the whitespace changes? I like when the diff is as small as possible :)